### ISSUE TITLE
Error en FadeIn en una llamada después de un dispose animate_do: ^3.3.0

### ISSUE THREAD

**Issue by larteagar** [Klerith/animate_do_package#32]
[ERROR:flutter/runtime/dart_vm_initializer.cc(41)] Unhandled Exception: 'package:flutter/src/animation/animation_controller.dart': Failed assertion: line 453 pos 7: '_ticker != null': AnimationController.forward() called after AnimationController.dispose()
AnimationController methods should not be used after calling dispose.
#0      _AssertionError._doThrowNew (dart:core-patch/errors_patch.dart:51:61)
#1      _AssertionError._throwNew (dart:core-patch/errors_patch.dart:40:5)
#2      AnimationController.forward (package:flutter/src/animation/animation_controller.dart:453:7)
#3      AnimateDoState.buildAnimation.<anonymous closure> (package:animate_do/src/types/animate_do_mixins.dart:64:32)
#4      new Future.delayed.<anonymous closure> (dart:async/future.dart:427:39)
#5      Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
#6      _Timer._runTimers (dart:isolate-patch/timer_impl.dart:398:19)
#7      _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:429:5)
#8    <…>
Restarted application in 766ms.

```
class _Slide extends StatelessWidget {
  final Movie movie;
  const _Slide({required this.movie});

  @override
  Widget build(BuildContext context) {
    final decoration = BoxDecoration(
        borderRadius: BorderRadius.circular(20),
        boxShadow: const [
          BoxShadow(
            color: Colors.black45,
            blurRadius: 10,
            offset: Offset(0, 10),
          )
        ]);
    return Padding(
      padding: const EdgeInsets.only(bottom: 30),
      child: DecoratedBox(
        decoration: decoration,
        child: ClipRRect(
          borderRadius: BorderRadius.circular(20),
          child: Image.network(
            movie.backdropPath,
            fit: BoxFit.cover,
            loadingBuilder: (context, child, loadingProgress) {
              if (loadingProgress != null) {
                return const DecoratedBox(
                  decoration: BoxDecoration(color: Colors.black12),
                );
              }
              return FadeIn(child: child);
            }
          ),
        ),
      ),
    );
  }
}
```

**Comment by Klerith** [comment]
Fue capaz  de reproducir el error, ya lo estoy investigando!

Estoy investigando, esto estará resuelto en el proximo "bug fix" release 3.3.1

**Comment by Klerith** [comment]
@larteagar arreglado en la versión 3.3.1 que ya está disponible
